common/src/test/java/tachyon/util/FormatUtilsTest.java

```diff
@@ -22,6 +30,147 @@
 
 public class FormatUtilsTest {
   @Test
+  public void parametersToStringTest() {
+    class TestCase {
+      String mExpected;
+      Object[] mInput;
+
+      public TestCase(String expected, Object[] objs) {
+        mExpected = expected;
+        mInput = objs;
+      }
+    }
+
+    List<TestCase> testCases = new LinkedList<TestCase>();
+    testCases.add(new TestCase("()", null));
+    testCases.add(new TestCase("(null)", new Object[] {null}));
+    testCases.add(new TestCase("()", new Object[] {""}));
```

Comment:
Could you add one more test case with multiple empty strings in the beginning of the array? Like {"", "", ""}?